user_account/api/serializers.py

```diff
@@ -15,3 +17,9 @@ def save(self):
         account.set_password(self.validated_data["password"])
         account.save()
         return account
+
+
+class AssignUserSerializer(serializers.ModelSerializer):
```

Comment:
I have issue with the name and the placement here. This is not user app related, but issue app, the name has verb in it, as if it meant to only do one thing, but in fact its `AssigneeSerializer`, as you time and time again use it in that context